src/Equinor.ProCoSys.IPO.Command/InvitationCommands/InvitationHelper.cs

```diff
@@ -15,6 +17,11 @@ public static string GetBaseUrl(string pcsBaseUrl, string plant)
 
         public static bool ParticipantIsSigningParticipant(ParticipantsForCommand participant) 
             => participant.Organization != Organization.External && participant.Organization != Organization.Supplier;
+        public static async Task<bool> HasIpoAdminPrivilege(IPermissionCache permissionCache, string plant, Guid currentUser)
```

Comment:
Not sure, but is there a common place where we can place this helper that can be reached in query as well? Where we wouldnt have to pass all of these parameters, but instead have them in constructor (IPlantProvider, ICurrentUserProvider, IPermissionCache)

Reply:
Or, this is only used from InvitationValidator? Maybe it is ok to just have it there?